reframe/core/fields.py

```diff
@@ -42,6 +42,9 @@ def __set__(self, obj, value):
 class ForwardField:
```

Comment:
I suggest removing completely the `ForwardField`. It's not used anywhere and it was a more verbose way for forwarding attribute access to a backend, which is now done much more easily with `__getattr__()`.

Reply:
Done